### PR TITLE
Updated to work with Redmine 3.3

### PR DESCRIPTION
**Pull request by shawndibble**
Redmine 3.3 adjusted their main-menu and added a drop down. These changes correct this issue.

### CODE REVIEW

File: src/sass/components/_top.scss [hunk 1]
```diff
@@ -214,10 +214,45 @@
...
+    display: none;
+    
+    &.visible {
+        display: block;
+        position: absolute;
+        background: #f5f5f5;
+        width: 130px;
+        z-index: 99;
+        margin: 0 !important;
+        padding: 0 !important;
```
Comment (mrliptontea) [inline src/sass/components/_top.scss]: are those `!important`s necessary?

Comment (shawndibble) [inline src/sass/components/_top.scss]: lines 226 and 248 (both margin: 0) can be removed. If I remove anything else, the layout screws up.

**Comment by archonwang** [comment]
@mrliptontea ,
It seems not worked in my environment. 
click the plus(+)button there's nothing happened.

I checked html code it seems contains the elements but I cannot see and click it.. How to solve it? Thanks.

**Comment by mrliptontea** [comment]
@archonwang try it now

**Pull request merged**